integration-tests/tests/src/tests/list.ts

```diff
@@ -666,7 +666,10 @@ describe("Lists", () => {
         );
         //@ts-expect-error can not assign an array to a list of objects.
         expect(() => (array[0] = array)).throws(Error, "Missing value for property 'TestObject.doubleCol'");
-        expect(() => (array[2] = { doubleCol: 1 })).throws(Error, "Requested index 2 greater than max 1");
+        expect(() => (array[2] = { doubleCol: 1 })).throws(
+          Error,
+          "Requested index 2 calling set() on list 'LinkTypesObject.arrayCol' when max is 1",
```

Comment:
In my opinion, this error message is less user friendly than the older one.  Maybe it's better for internal developers.